Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -156,3 +156,180 @@ def __enter__(self):
     return slicer.dicomDatabase
   def __exit__(self, type, value, traceback):
     closeTemporaryDatabase(self.originalDatabaseDir)
+
+#------------------------------------------------------------------------------
+# Download file to local storage.
+# Target file path needs to indicate the file name and extension as well
+def downloadFile(url, targetFilePath):
+  if not os.path.exists(targetFilePath) or os.stat(targetFilePath).st_size == 0:
+    logging.info('Downloading from\n  %s\nas file\n  %s\nIt may take a few minutes...' % (url,targetFilePath))
+    try:
+      import urllib
+      urllib.urlretrieve(url, targetFilePath)
+    except Exception, e:
+      import traceback
+      traceback.print_exc()
+      logging.error('Failed to download file from ' + url)
+      return False
+  else:
+    logging.info('Requested file has been found: ' + targetFilePath)
+  return True
+
+#------------------------------------------------------------------------------
+# Unzip file into folder. Number of expected files unzipped may be specified.
+# If folder contains the same number of files as expected (if specified), then
+#   it will be assumed that unzipping has been successfully done earlier.
+def unzipFile(zipFilePath, unzipDir, expectedNumberOfUnzippedFiles=0):
```

Comment:
consider setting `expectedNumberOfUnzippedFiles=None` if you want to ignore the value.

Reply:
0 seemed to make sense because nobody wants to extract 0 files. Is None a convention to use for the abscence of an argument value?